lib/fathead/django_built_ins/output.txt

```diff
@@ -0,0 +1,291 @@
+autoescape	A							https://www.djangoproject.com/			<pre><code>{% autoescape on %}
```

Comment:
Each entry needs to be on a single line. You can use `<br>` in stead of newlines.